tests/index.test.js

```diff
@@ -45,7 +42,7 @@ describe('Action', () => {
       'proxy-port': '9999',
       'proxy-scheme': 'https',
     });
-    expect(clientInitSpy).toHaveBeenCalledWith(
+    expect(initSpy).toHaveBeenCalledWith(
```

Comment:
If we used this in more than one test we would need to use reset/restore all mocks, but this is fine for the tests as they are.